css/properties/position-visibility.json

```diff
@@ -0,0 +1,40 @@
+{
+  "css": {
+    "properties": {
+      "position-visibility": {
```

Comment:
Missing sub features: always  anchors-valid, anchors-visible, no-overflow

Reply:
added in next commit

Reply:
OK, got it. I've removed it from the data for now.